extractor/src/main/java/org/schabi/newpipe/extractor/stream/AudioTrackType.java

```diff
@@ -0,0 +1,33 @@
+package org.schabi.newpipe.extractor.stream;
+
+/**
+ * An enum representing the track type of an {@link AudioStream} extracted by a {@link
+ * StreamExtractor}.
+ */
+public enum AudioTrackType {
+    /**
+     * The original audio track of the video
```

Comment:
While that's not the case currently, maybe one day audio services we support and/or add could support multiple audio tracks, so I would use `stream` instead of `video` here.

Also, as multiple original audio tracks exist for some services (YouTube for instance), I would use an indefinite article.

```suggestion
     * An original audio track of the stream.
```

For consistency with most of the JavaDocs, please also add a period at the end of the title of the ones you added.